geps/gep-1897.md

```diff
@@ -125,7 +126,235 @@ This GEP is the outcome of the TLS use cases #4 and #5 in
 
 ## API
```

Comment:
We talked about this one in the community sync today: we want to at least acknowledge mTLS use cases as a part of this doc, even if it is ultimately to say they wont be in scope (and why). We discussed making this an "unresolved" / "todo" so that it doesn't have to hold up this specific PR but it does need to hold us from moving out of `Provisional` state.

Reply:
Please see https://github.com/kubernetes-sigs/gateway-api/pull/2113/files/5d84379d2aa570958ce6dbd3fb8cf187447e9f95#r1231560568 and let me know if that is the discussion that needs to be a part of the proposal.

I have documented the mTLS use case are longer term goals that require more attention than this GEP could provide.  https://github.com/kubernetes-sigs/gateway-api/pull/2113/files/5d84379d2aa570958ce6dbd3fb8cf187447e9f95#diff-fbb3fdd16d81d10c31ffcac7b57c3107034f697fcf1c7b291f5d9e7ec51a4125L26

Reply:
In the next update I will also push a longer explanation in the "Longer Term Goals" section.